### PR TITLE
Add turkish translations

### PR DESCRIPTION
**Pull request by mikropsoft**

### CODE REVIEW

**Comment by msasikanth** [comment]
@mikropsoft can you please run code formatting once? You can run this Gradle command `gradle spotlessApply`

File: resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/TrTwineStrings.kt [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sasikanth.rss.reader.resources.strings
+
+import cafe.adriel.lyricist.LyricistStrings
+
+@LyricistStrings(languageTag = Locales.TR, default = true)
```
Comment (msasikanth) [inline resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/TrTwineStrings.kt]: Please remove the default value here.

**Review by msasikanth — CHANGES_REQUESTED** [review]

File: resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/TrTwineStrings.kt (hunk 1 above)
Comment (mikropsoft) [reply resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/TrTwineStrings.kt]: changed the value to "false"

**Comment by msasikanth** [comment]
@mikropsoft there are duplicate commits and merge commit. Can you please clean up the PR commits, so that I can merge this in, please?

**Comment by mikropsoft** [comment]
> @mikropsoft there are duplicate commits and merge commit. Can you please clean up the PR commits, so that I can merge this in, please? 

I don't know exactly how to do this. Do you want me to reopen Pr

**Comment by msasikanth** [comment]
> > @mikropsoft there are duplicate commits and merge commit. Can you please clean up the PR commits, so that I can merge this in, please?
> 
> I don't know exactly how to do this. Do you want me to reopen Pr

Yes, please.

**Comment by mikropsoft** [comment]
Sure

**Pull request closed without merging**